arch/arm/src/stm32f0l0g0/stm32_tim.c

```diff
@@ -897,11 +897,13 @@ static int stm32_tim_setmode(struct stm32_tim_dev_s *dev,
    * disable it, simply set its clock to valid frequency or zero.
    */
 
+#ifdef STM32_TIM6_BASE
```

Comment:
So, all devices which have TIM6 are supposed to have TIM7 as well?

Reply:
actually no, some stm32l0 have only TIM6, no TIM7